app/controllers/cookies_controller.rb

```diff
@@ -0,0 +1,39 @@
+class CookiesController < ApplicationController
+  def accept_analytics
+    write_cookie(:cookies_policy, :analytics_accepted)
+    redirect_back fallback_location: root_path
+  end
+
+  def reject_analytics
+    write_cookie(:cookies_policy, :analytics_rejected)
+    redirect_back fallback_location: root_path
+  end
+
+  def hide_this_message
+    write_cookie(:cookies_preferences_set, true)
+    redirect_back fallback_location: root_path
+  end
+
+  def update
+    cookies.to_hash.each_pair { |k, _v| cookies.delete k, domain: ".#{request.domain}" }
```

Comment:
Not sure this is the best way to delete the ga cookies. Currently, on dev, they are write to a wild card to the domain '.fra-dev.aws.defra.cloud'.

Reply:
Update: this still doesn't delete the ga cookies on dev. Need to have a re-think

Reply:
This now works as ".#{request.hostname}". Have checked that GA cookies on production are also create with the domain: ".<hostname>"